test/e2e/framework/resource/resources.go

```diff
@@ -175,6 +175,26 @@ func waitForPodsInactive(ps *testutils.PodStore, interval, timeout time.Duration
 	return err
 }
 
+// filterActivePods returns pods that have not terminated.
+func filterActivePods(pods []*v1.Pod) []*v1.Pod {
```

Comment:
This function seems to only be used (in k/k/test) here.
Though given the name and the fact that it uses `IsPodActive()` (which would definitely fit in into the e2epod pkg), I wonder whether we should keep it here, e2eresource, or move it over to e2epod.
I vote for e2epod just based on the sort of logic.

Reply:
+1  for e2epod 😄